src/go/rpk/pkg/config/params.go

```diff
@@ -646,6 +651,15 @@ func (p *Params) InstallCloudFlags(cmd *cobra.Command) {
 	cmd.MarkFlagsRequiredTogether(FlagClientID, FlagClientSecret)
 }
 
+func (p *Params) InstallFormatFlag(cmd *cobra.Command) {
+	pf := cmd.PersistentFlags()
+
+	pf.StringVar(&p.Formatter.kind, "format", "text", fmt.Sprintf("Output format (%v)", strings.Join(supportedFormats, ",")))
```

Comment:
Perhaps a way to do this would be to expose a field
```go
type Formatter struct {
    ExtraText []string
}
```
and then isText matches for either `text` or any of `ExtraText`

Reply:
>We'll need the formatter to support more than just text in the future as well -- some commands support short/text and long/wide at the moment.

I think this supports that, we are just using the default as `text`, How is different from having another "kind" like `short` and `wide` in the list since they are all different formats that are not meant to be together? 

or do you mean that we can have a combination of `wide|short` and `json`/`text` ?

I don't get what's the `isText` in this context.

Reply:
As we align internally, I added support for `wide` and `long`, and exposed a new boolean `isLong`